internal/server/storage/drivers/driver_zfs.go

```diff
@@ -261,7 +280,14 @@ func (d *zfs) Create() error {
 		}
 
 		// Create the zpool.
-		_, err = subprocess.RunCommand("zpool", "create", "-m", "none", "-O", "compression=on", d.config["zfs.pool_name"], loopPath)
+		createArgs := []string{"create", "-m", "none", "-O", "compression=on", d.config["zfs.pool_name"]}
+		// "zpool create" doesn't have an explicit type for "stripe" vdev type
+		if vdevType != zfsDefaultVdevType {
+			createArgs = append(createArgs, vdevType)
+		}
+
+		createArgs = append(createArgs, loopPath)
+		_, err = subprocess.RunCommand("zpool", createArgs...)
```

Comment:
Hmm, I thought I pointed that one out already :)

This is the `loopPath` case, this cannot have more than a single device which is the loopPath so there is no need to change anything to the current logic.

Reply:
I've updated `FillConfig` to ensure that if there are multiple devices, all of them have to be existing block devices.

Reply:
Ah good, that makes sense